packages/world/src/modules/core/CoreModule.sol

```diff
@@ -58,6 +59,7 @@ contract CoreModule is IModule, WorldContextConsumer {
    */
   function _registerCoreTables() internal {
     InstalledModules.register();
+    Delegations.register();
```

Comment:
any easy way to add some tests to ensure we're registering all of our expected core tables?

Reply:
no automated way, maintaining the test would have the same issue as maintaining the actual registration

Reply:
well we could add some sort of e2e test that checks that all tables in the world config are registered, but for that we first need to separate the test tables

Reply:
opened an issue to track here: https://github.com/latticexyz/mud/issues/1456